public/index.html

```diff
@@ -26,11 +27,24 @@
     <meta
       property="og:image"
       name="twitter:image"
-      content="%PUBLIC_URL%/og-image.png"
+      content="https://www.excalidraw.com/og-image.pngg"
     />
     <meta property="og:image:width" content="1280" />
     <meta property="og:image:height" content="669" />
 
+    <!-- Twitter Card tags -->
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:creator" content="@Vjeux" />
```

Comment:
Let's get rid of this line. At this point it's truly a team effort and not sure it's worth calling me out directly.

Reply:
Agreed - team work makes the dream work 🎉